predicators/spot_utils/spot_utils.py

```diff
@@ -421,13 +423,58 @@ def execute(self, name: str, objects: Sequence[Object],
                 params: Array) -> None:
         """Run the controller based on the given name."""
         assert self._connected_to_spot
+        if name == "find":
+            self._find_controller_move_queue_idx += 1
+            return self.findController()
+        # Just finished finding.
+        self._find_controller_move_queue_idx = 0
+        if name == "stow":
+            return self.stow_arm()
         if name == "navigate":
             return self.navigateToController(objects, params)
         if name == "grasp":
             return self.graspController(objects, params)
         assert name == "placeOnTop"
         return self.placeOntopController(objects, params)
 
+    def findController(self) -> None:
+        """Execute look around."""
+        # Execute a hard-coded sequence of movements and hope that one of them
+        # puts the lost object in view. This is very specifically designed for
+        # the case where an object has fallen off the table and we are facing
+        # the table when this controller starts running.
+
+        # Start by stowing.
+        self.stow_arm()
+
+        # Move back from the table.
+        if self._find_controller_move_queue_idx == 1:
```

Comment:
This is a great start, but I imagine we want to later implement separate things for when we want to find from the rack/other places than the table.